src/app/lib/config/services/korean.js

```diff
@@ -146,12 +145,16 @@ const service = {
           href: 'https://www.bbc.co.uk/usingthebbc/terms/',
           text: '이용 약관',
         },
+        {
+          href: 'https://www.bbc.co.uk/aboutthebbc',
+          text: 'BBC 소개',
+        },
         {
           href: 'https://www.bbc.co.uk/usingthebbc/privacy/',
           text: '개인정보취급방침',
         },
         {
-          href: 'https://www.bbc.co.uk/usingthebbc/cookies/',
+          href: 'https://www.bbc.com/usingthebbc/cookies/',
```

Comment:
🤔 Any reason this link point to a `.com` link while the rest of them are `.co.uk` links? These URLs have similar content.

Reply:
I must have accidentally changed it to a `.com` while I was updating the links. Reverted that. Though since it's an international service, it would redirect to a `.com`